zulipterminal/core.py

```diff
@@ -298,15 +300,16 @@ def show_all_mentions(self, button: Any) -> None:
                         anchor=None,
                         mentioned=True)
 
-    def view_in_browser(self, message_id: int) -> None:
-        url = '{}#narrow/near/{}'.format(self.model.server_url, message_id)
+    def view_in_browser(self, message: Message) -> None:
+        # Truncate extra '/' from the server url.
+        self.url = near_message_url(self.model.server_url[:-1], message)
```

Comment:
It's good to use the pre-existing code here, though using `urljoin` in the new code would mean we don't need to worry about this. I'm guessing the server code avoids this since it knows the url exactly and relies on tests, which we should too really :)